src/Model/UserManager.php

```diff
@@ -4,8 +4,10 @@
 
 class UserManager extends AbstractManager
 {
-    public const TABLE = 'user';
-
+    public const TABLE = 'comic_book';
```

Comment:
No ! You are in UserManager so your TABLE should be "user" not "comic_book". And why your TABLE2 is the same as your TABLE4 ?? Clean that part